Error on first run: TypeError: Cannot read property 'invalid' of undefined

Getting an error after installing

TypeError: Cannot read property 'invalid' of undefined

```
$ npm run cosmos

> edc-web@0.1.0 cosmos /home/ettinger/projects/profullstack/edcast/edc-web
> cosmos

[Cosmos] See you at http://localhost:5000
[Cosmos] Using webpack config found at webpack.config.js
[Cosmos] Server crashed...

  (╯°□°)╯︵ ┻━┻

TypeError: Cannot read property 'invalid' of undefined
    at _callee2$ (/home/ettinger/projects/profullstack/edcast/edc-web/node_modules/react-cosmos/dist/plugins/webpack/devServer.js:66:35)
    at tryCatch (/home/ettinger/projects/profullstack/edcast/edc-web/node_modules/regenerator-runtime/runtime.js:45:40)
    at Generator.invoke [as _invoke] (/home/ettinger/projects/profullstack/edcast/edc-web/node_modules/regenerator-runtime/runtime.js:271:22)
    at Generator.prototype.(anonymous function) [as next] (/home/ettinger/projects/profullstack/edcast/edc-web/node_modules/regenerator-runtime/runtime.js:97:21)
    at asyncGeneratorStep (/home/ettinger/projects/profullstack/edcast/edc-web/node_modules/react-cosmos/dist/plugins/webpack/devServer.js:22:103)
    at _next (/home/ettinger/projects/profullstack/edcast/edc-web/node_modules/react-cosmos/dist/plugins/webpack/devServer.js:24:194)
    at /home/ettinger/projects/profullstack/edcast/edc-web/node_modules/react-cosmos/dist/plugins/webpack/devServer.js:24:364
    at new Promise (<anonymous>)
    at /home/ettinger/projects/profullstack/edcast/edc-web/node_modules/react-cosmos/dist/plugins/webpack/devServer.js:24:97
    at _webpackDevServer (/home/ettinger/projects/profullstack/edcast/edc-web/node_modules/react-cosmos/dist/plugins/webpack/devServer.js:148:28)
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! edc-web@0.1.0 cosmos: `cosmos`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the edc-web@0.1.0 cosmos script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/ettinger/.npm/_logs/2019-12-10T21_03_36_033Z-debug.log
```

--- a/README.md
+++ b/README.md
@@ -107,7 +107,7 @@ _Something wrong?_ Don't hesitate to [create a GitHub issue](https://github.com/
 
 The only hard requirements are React 16.8 and Node 6 (or newer).
 
-React Cosmos works best with webpack. It takes extra effort to make it work with other bundlers, but it's not as scary as it might seem. Don’t be afraid to ask for support.
+React Cosmos works best with webpack 4. It takes extra effort to make it work with other bundlers, but it's not as scary as it might seem. Don’t be afraid to ask for support.
 
 > [Browserify](https://github.com/react-cosmos/react-cosmos-classic/tree/14e1a258f746df401a41ab65429df0d296b910e4/examples/browserify) and [Parcel](https://github.com/react-cosmos/parcel-ts-example) examples are available for Cosmos Classic. Props to whoever adapts them to React Cosmos 5!
 

